src/libertem/executor/dask.py

```diff
@@ -186,6 +186,33 @@ def get_available_workers(self):
             for worker in info['workers'].values()
         ])
 
+    def get_resource_details(self):
+        workers = self.get_available_workers()
+        details = {}
+
+        for worker in workers:
+            if worker.name.startswith("tcp"):
+                host_name = worker.host
+                resource = 'cpu'
+            else:
+                host_name = '-'.join(worker.name.split('-')[:-2])
+                resource = worker.name.split('-')[-2]
```

Comment:
The `Worker` class should have access to `host` and the `resources` list directly - is there a reason this is not used here?

Reply:
`host` is IP address, should I add an extra attribute to worker class for `host_name`?

Reply:
![image](https://user-images.githubusercontent.com/59721339/96957761-3277a680-1519-11eb-914f-310824b2076e.png)

I think it's better to use host IP address instead of host name, in the above example both `dask-worker` and `libertem-worker` is running on same device. But currently, it's showing as 2 hosts.

Reply:
Agreed, consistency is important! :+1:

Reply:
> I think it's better to use host IP address

Needs updated test case (maybe just check all other fields than `host`, as that will vary across runs)